rts/Game/Camera/OverviewController.h

```diff
@@ -9,9 +9,18 @@ class COverviewController : public CCameraController
 {
 public:
 	COverviewController();
+	~COverviewController();
+
+	constexpr static float3 DIR_UP = float3(0.0f, -0.999999500000375f, -0.000999999500000375f); // -90 deg
```

Comment:
looks suspiciously close to `(0, -1, 0)`, could use a comment on why the tiny difference

Reply:
That's a good question! This is the explicit version of what was the previous dir `float3(0.0f, -1.0f, -0.001f).ANormalize()`.

I have absolutely no idea!

Reply:
Try `(0, -1, 0)` and see what happens.

AFAIK it's just to specify a "heading" around the Y axis (i.e. otherwise it's unspecified where "north" is, for example for camera panning purposes) but hopefully that's where the other constants (`DIR_{LEFT,RIGHT}`) come in.

A straight top-down angle also used to break shadows in the past, could check if that is still the case.